modules/weko-admin/weko_admin/models.py

```diff
@@ -776,6 +776,87 @@ def __iter__(self):
                     yield (name, value)
 
 
+class BillingPermission(db.Model):
+    """Database for Billing Permission."""
+
+    __tablename__ = 'billing_permission'
+
+    user_id = db.Column(
+        db.Integer(),
+        primary_key=True,
+        nullable=False, 
+        unique=True
+    )
+
+    is_active = db.Column(
+        db.Boolean(name='active'),
+        default=True, 
+        nullable=False
+    )
+
+    @classmethod
+    def create(cls, user_id, is_active=True):
+        """Create new user can access billing file.
+
+        :param user_id: user's id
+        :param is_active: access state
+        :return: Unit if create succesfully
+        """
+        try:
+            obj = BillingPermission()
+            with db.session.begin_nested():
+                obj.user_id = user_id
+                obj.is_active = is_active
+                db.session.add(obj)
+            db.session.commit()
+        except BaseException as ex:
+            db.session.rollback()
+            current_app.logger.debug(ex)
+            raise
+
+        return cls
+
+    @classmethod
+    def activation(cls, user_id, is_active):
+        """Change access state of user.
+
+        :param user_id: user's id
+        :param is_active: access state
+        :return: Updated records
+        """
+        try:
+            with db.session.begin_nested():
+                billing_data = cls.query.filter_by(
+                    user_id=user_id).one_or_none()
+                if billing_data:
+                    billing_data.is_active = is_active
+                    db.session.merge(billing_data)
+                else:
+                    cls.create(user_id, is_active)
+                    current_app.logger.debug('New user is created!')
```

Comment:
should be using "current_app.logger.info()"

Reply:
Thank for your comment.
It had been fix at last rev.